Wikia/selenium-tests
# Xw 196

Test steps:
 1. Open mercury empty CC editor wikia in edit mode
 2. click add new feature
 3. click add image
 4. generate new image
 5. upload the image
 6. make sure flow taken you to the cropping tool
 7. click done - being able to do this indicates that all went fine
